docs/application/native/guides/app-management/app-controls.md

```diff
@@ -457,17 +462,17 @@ You can allow other applications to launch your application and use your applica
 >
 > In the application manifest file, the valid operation name format is `http://tizen.org/appcontrol/operation/<verb>`. You cannot use the related macro name, `APP_CONTROL_OPERATION_<VERB>`.
 
-The operation, URI, and MIME type information is used when [resolving the application control](#resolution). The operation information is mandatory, while the URI or MIME type information is optional. Any application requesting a launch of your application must either specify your application ID (for an explicit launch) or have the same operation value and applicable URI and MIME type information (for an implicit launch).
+The operation, URI, and MIME type information is used when [resolving the application control](#resolution). The operation information is mandatory, while the URI or MIME type information is optional. Any application requesting of your application must either specify your application ID (for an explicit launch) or have the same operation value and applicable URI and MIME type information (for an implicit launch).
 You can define privileges to restrict your application from launching. To launch your application, the caller application must request for launch permission by defining privileges in the `tizen-manifest.xml` file.
 
 > **Note**
 >
 > The URI or MIME type can contain wildcards, such as '\*', to match against given conditions in the app control:
 >
 > - In the MIME type, you can use two types of wildcards: `image/*` and `*/*`.
-> - In the URI, a more complex pattern of wildcards with similar semantics as the standard `glob()` function is available: '\*' matches an arbitrary, possibly empty, string, and '?' matches an arbitrary character. Unlike in the `glob()` function, the '/' character can be matched by the wildcards. There are no [...] character ranges, and the wildcards '\*' and '?' cannot be escaped to include them literally in a pattern.
+> - In the URI, a more complex pattern of wildcards with similar semantics as the standard `glob()` function is available, where '\*' matches an arbitrary, possibly empty, string, and '?' matches an arbitrary character. Unlike in the `glob()` function, the '/' character can be matched by the wildcards. There are no [...] character ranges, and the wildcards '\*' and '?' cannot be escaped to include them literally in a pattern.
 
-You can specify the application control information for your application in the [application project settings](../../tutorials/process/setting-properties.md#manifest) in the Tizen Studio.
+You can specify the control information from [application project settings](../../tutorials/process/setting-properties.md#manifest).
```

Comment:
@mijin85cho I checked it and this is also grammatically correct whereas it can be rephrased as
"You can specify the control information in the [application project settings](../../tutorials/process/setting-properties.md#manifest)". Please let me know for the suggestion.

Reply:
@rishu-samsung One of the main reason we ask review to SRI-B is to check the grammar. Please always suggest grammatically correct sentences.